api/machine/machine.proto

```diff
@@ -47,6 +47,8 @@ service MachineService {
   rpc Rollback(RollbackRequest) returns (RollbackResponse);
   rpc Reset(ResetRequest) returns (ResetResponse);
   rpc Recover(RecoverRequest) returns (RecoverResponse);
+  rpc RemoveBootkubeInitializedKey(google.protobuf.Empty)
```

Comment:
temporary API to drop `initialized` key from etcd (as a signal that no longer running self-hosted control plane)